uber/templates/emails/shifts/schedule.html

```diff
@@ -5,7 +5,7 @@
 {{ attendee.first_name }},
 <br/> <br/>
 
```

Comment:
Haha, I'd forgotten about that timezone bug.  Stupid MagWest and its stupid different timezone making us care about stupid timezones :P